contrib/buildsystems/CMakeLists.txt

```diff
@@ -35,17 +40,16 @@ empty(default) :
 NOTE: -DCMAKE_BUILD_TYPE is optional. For multi-config generators like Visual Studio
 this option is ignored
 
-This process generates a Makefile(Linux/*BSD/MacOS) , Visual Studio solution(Windows) by default.
+This process generates: a Makefile(Linux/*BSD/MacOS), Visual Studio solution(Windows) by default.
```

Comment:
Please remove the colon, it makes at least this reader stumble all over this sentence.

Reply:
done